corehq/apps/importer/util.py

```diff
@@ -121,6 +121,17 @@ def __init__(self, file_path, column_headers):
         except Exception:
             self.has_errors = True
 
+    def _col_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.col(index)]
+
+    def _row_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.row(index)]
+
+    def _fmt_value(self, cell):
+        if cell.ctype == XL_CELL_NUMBER and int(cell.value) == cell.value:
```

Comment:
Yeah, the cell is already an integer, but xlrd treats all numbers like decimals: if the spreadsheet contains "1", the built-in `row_values` and `col_values` functions will return it as "1.0".